components/scream/src/share/io/scorpio_input.cpp

```diff
@@ -123,6 +123,9 @@ set_grid (const std::shared_ptr<const AbstractGrid>& grid)
   EKAT_REQUIRE_MSG (grid->is_unique(),
       "Error! I/O only supports grids which are 'unique', meaning that the\n"
       "       map dof_gid->proc_id is well defined.\n");
+  EKAT_REQUIRE_MSG (
```

Comment:
I think I know the answer, but might as well ask.  How will this work for the future when we might want non-contiguous grids.  I.e. twin column.

Reply:
The contiguity is at the _global_ level, not necessarily local. Even now, with 2+ ranks, the physics GLL grid does not have contiguous column gids.

Another way to look at this, is that running with 1 rank you must get gids `[g0,...,g0+num_dofs)`.

Reply:
ahhh okay, now I understand.  Great.